tests/test_vec_envs.py

```diff
@@ -0,0 +1,36 @@
+import gym
+from stable_baselines.common.vec_env.dummy_vec_env import DummyVecEnv
+from stable_baselines.common.vec_env.subproc_vec_env import SubprocVecEnv
+
+ENV_ID = 'BreakoutNoFrameskip-v4'
+
+
+def test_dummyvecenv():
```

Comment:
pytest provide a @pytest.mark.parametrize to avoid duplication, see https://github.com/hill-a/stable-baselines/blob/master/tests/test_identity.py#L30 for an example ;)